### ISSUE TITLE
snapshot creation vs restore with the cli uses different names?

### ISSUE THREAD

**Issue by cogneato** [home-assistant/hassio-cli#75]
using `hassio snapshots new --name "snapshotName"` creates a full snapshot, but when restoring a snapshot the name has to be the snapshot's slug name. Is this the way it is supposed to be?

**Comment by poldim** [comment]
I found out when trying to restore through trial and error that it uses the `slug` instead of the `name` as indicated in the help for `hassio snapshots`:

```NAME:
   hassio snapshots - list, info, reload, new, restore, remove

USAGE:
   hassio snapshots [command options] [arguments...]

OPTIONS:
   --rawjson, -j                                      Returns the output in JSON format
   --options key=val,key2=val2, -o key=val,key2=val2  holds data for POST in format key=val,key2=val2
   --filter prop1,prop2, -f prop1,prop2               properties to extract from returned data prop1,prop2
   --name nameofsnapshot                              used with 'info|remove|restore|full' actions to return info on a specific snapshot nameofsnapshot
```

**Comment by poldim** [comment]
‘VERSION:
   1.2.1’

DK

**Comment by ryanm101** [comment]
Should be fixed in 1.3.

@frenck @pvizeli are the new ssh addons out with the updated CLI?
